[Reporting] Kibana Application Privileges for Reporting (#94966)
elastic/kibana: docs/user/reporting/index.asciidoc

[[reporting-required-privileges]]
== Roles and privileges

> When security is enabled, access to reporting features is controlled by security privileges. In versions 7.12 and below, you can do this

Reviewer: ```suggestion
When security is enabled, access to the {report-features} is controlled by security privileges. In versions 7.12 and earlier, you can grant access to the {report-features}
```

> by assigning users the `reporting_user` role in Elasticsearch. In 7.13 and above, you can configure Reporting to use

Reviewer: ```suggestion
by assigning users the `reporting_user` role in {es}. In 7.13 and later, you can configure *Reporting* to use
```

> <<kibana-privileges, {kib} privileges>> instead. It is recommended that Reporting is configured to

Reviewer: ```suggestion
<<kibana-privileges, {kib} privileges>>. It is recommended that *Reporting* is configured to
```

> use Kibana privileges by setting `xpack.reporting.roles.enabled` to `false` in kibana.yml. By using Kibana privileges, you can define

Reviewer: ```suggestion
use {kib} privileges by setting <<xpack-reporting-roles-enabled,`xpack.reporting.roles.enabled`>> to `false`. By using {kib} privileges, you can define
```

> custom roles that grant Reporting privileges as sub-features of Kibana applications in Role Management.

Reviewer: ```suggestion
custom roles that grant *Reporting* privileges as sub-features of {kib} applications in *Role Management*.
```

> Users also need the appropriate {kib} privileges to access the objects that you

Reviewer: ```suggestion
Users must also have the {kib} privileges to access the saved objects and associated {es} indices included in the generated reports.
```

> want to report on and the {es} indices.  See <<secure-reporting, Reporting and

Reviewer: ```suggestion
For an example, refer to <<secure-reporting, Reporting and
```

> security>> for an example.

Reviewer: ```suggestion
security>>.
```
